modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/containers/FormNoop.es.js

```diff
@@ -12,24 +12,21 @@
  * details.
  */
 
-import {PagesVisitor} from '../../util/visitors.es';
+import React from 'react';
 
-export default (pages, properties) => {
-	const {fieldInstance} = properties;
-	const pageVisitor = new PagesVisitor(pages);
+import Pages from '../components/Pages.es';
+import {FormNoopProvider} from '../hooks/useForm.es';
 
-	return Promise.resolve(
-		pageVisitor.mapFields((field) => {
-			const matches =
-				field.name === fieldInstance.name &&
-				field.required &&
-				fieldInstance.value == '';
+/**
+ * It is an implementation of Form no-op, it just renders the
+ * layout and propagates any event to the instance `dispatch`.
+ */
```

Comment:
I'd also be interested in hearing ideas for a better name here. I know @brunobasto said it sounded a bit off to him and I agree, although I that might be because I don't understand what it's really for.

Reply:
Oh no 🥺. That sounds like a good name to me after removing the word `Renderer` from the middle. haha 

A Form has reducers, in this case, Noop does not make any internal state changes that is, we remove the reducers. Only the spread of events.